CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Fish is now more forgiving of missing or invalid $TERM values (#3850).
 - The `string` command now supports a `repeat` subcommand with the obvious behavior (#3864).
 - The `string match` command now supports a `--filter` flag to emit the entire string partially matched by a pattern (#3957).
+- The `string replace` command now supports a `--filter` flag to limit output to strings which underwent a replacement (#3348).
```

Comment:
Scratch that. I'm going to replace `-f`/`--filter` on `string match` with `-e`/`--entire`. Better to avoid any confusion about the `-f` short flag.